components/instantiation/NetPyNEInstantiated.js

```diff
@@ -142,62 +180,41 @@ export default class NetPyNEInstantiated extends React.Component {
             controls = (
                 <Menu>
-                    <MenuItem id={"tracesPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="Cell traces" onClick={() => { this.plotFigure('Traces Plot', 'plotTraces') }} />
-                    <MenuItem id={"rasterPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="Raster plot" onClick={() => { this.plotFigure('Raster Plot', 'plotRaster') }} />
-                    <MenuItem id={"spikePlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="Spike histogram" onClick={() => { this.plotFigure('Spike Hist Plot', 'plotSpikeHist') }} />
-                    <MenuItem id={"spikeStatsPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="Spike stats" onClick={() => { this.plotFigure('Spike Stats Plot', 'plotSpikeStats') }} />
-                    <MenuItem id={"ratePSDPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="Rate PSD" onClick={() => { this.plotFigure('Rate PSD Plot', 'plotRatePSD') }} />
-                    <MenuItem id={"LFPTimeSeriesPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="LFP time-series" onClick={() => { this.plotFigure('LFP Time Series Plot', 'plotLFP', 'timeSeries') }} />
-                    <MenuItem id={"LFPPSDPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="LFP PSD" onClick={() => { this.plotFigure('LFP PSD Plot', 'plotLFP', 'PSD') }} />
-                    <MenuItem id={"LFPSpectrogramPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="LFP spectrogram" onClick={() => { this.plotFigure('LFP Spectrogram Plot', 'plotLFP', 'spectrogram') }} />
-                    <MenuItem id={"LFPLocationsPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="LFP locations" onClick={() => { this.plotFigure('LFP Locations Plot', 'plotLFP', 'locations') }} />
-                    <MenuItem id={"grangerPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="Granger causality plot" onClick={() => { this.plotFigure('Granger Plot', 'granger') }} />
-                    <MenuItem id={"rxdConcentrationPlot"} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText="RxD concentration plot" onClick={() => { this.plotFigure('RxD concentration plot', 'plotRxDConcentration') }} />
+                    {plots.map((el, index) => {return <MenuItem id={"el.id"} key={index} style={styles.menuItem} innerDivStyle={styles.menuItemDiv} primaryText={el.primaryText} onClick={() => { this.plotFigure(el.plotName, el.plotMethod, el.plotType)}} />})}
                 </Menu>
             );
         }
 
         return (
-            <div id="instantiatedContainer" style={{ height: '100%', width: '100%' }}>
+            <div id="instantiatedContainer" style={{ height: '100%', width: '100%', position: 'fixed' }}>
```

Comment:
could the   >> position: fixed   <<  being causing the performance issues? I tried something different but nothing

Reply:
why do you think that?

Reply:
Because as Arthur Conan Doyle wrote: "When you have eliminated the impossible, whatever remains, however improbable, must be the truth"
I don't know, I have tried many things and nothing seems to remove the performance issue.

Reply:
I was asking you because there are some performing issues related to the use of dicts and object inside the render method (somebody told me a few days ago). I don't think this is our problem (or not only this) but for sure something to look at. Anyway, probably better to merge this and look at it as part of another card/PR